test-main.js

```diff
@@ -12,7 +12,7 @@ System.config({
   defaultJSExtensions: true,
   paths: {
     'angular2/*': 'node_modules/angular2/*.js',
-    '@reactivex/rxjs/*': 'node_modules/@reactivex/rxjs/*.js',
+    '@reactivex/rxjs/*': 'node_modules/angular2/node_modules/@reactivex/rxjs/*.js'
```

Comment:
Are you using npm 3.x.x? It is one of the requirements for the project, I think that with the flat module structure of it this change is not required.